### PR TITLE
test1

### PR DESCRIPTION
**Pull request by kerbecs**
New tests

### CODE REVIEW

**Review by edic21 — COMMENTED** [review]
more tests

File: tests.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Test1
```
Comment (edic21) [inline tests.txt]: more tests

**Comment by kerbecs** [comment]
oke

File: tests.txt (hunk 1 above)
Comment (edic21) [reply tests.txt]: at least 2

Comment (kerbecs) [reply tests.txt]: ok

File: tests.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+test2
```
Comment (edic21) [inline tests.txt:2]: capital letter

**Review by edic21 — CHANGES_REQUESTED** [review]
maybe a 4th test

**Review by edic21 — APPROVED** [review]

**Review by edic21 — APPROVED** [review]

**Review by edic21 — APPROVED** [review]

**Review by edic21 — APPROVED** [review]

File: tests.txt [hunk 3]
```diff
@@ -1,3 +1,4 @@
 Test1
 test2
-test3
\ No newline at end of file
+test3
```
Comment (edic21) [inline tests.txt:3]: hmmmm

Comment (kerbecs) [reply tests.txt:3]: what's wrong?

**Review by edic21 — COMMENTED** [review]
???

**Pull request closed without merging**